src/commands/server.ts

```diff
@@ -9,14 +9,26 @@ export const serverCommand = new Command('server')
   .option('--car-storage <path>', 'path for CAR file storage', './cars')
   .option('--database <path>', 'path to SQLite database', './pins.db')
   .option('--private-key <key>', 'private key for Synapse (or use PRIVATE_KEY env var)')
-  .option('--rpc-url <url>', 'RPC URL for Filecoin network', RPC_URLS.calibration.websocket)
+  .option(
+    '--network <network>',
+    'Filecoin network to use: mainnet or calibration (can also use NETWORK env)',
+    'calibration'
+  )
```

Comment:
import above with `import { Command, Option } from 'commander'`

```suggestion
  .addOption(
    new Option('--network <network>', 'Filecoin network to use (can also use NETWORK env)')
      .choices(['mainnet', 'calibration'])
      .default('calibration')
  )
```